base.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
+    "plugin: import/order",
```

Comment:
@SarahHouben there is a space there, looks wrong no?

Reply:
True! Removed the space.